tools/spice/spice_examples.py

```diff
@@ -60,19 +64,22 @@ def get_attitude_timerange(ck_kernel, id):
 
 
 def _get_particle_velocity(direct_events):
-    """Get the particle velocity in the heliosphere frame.
+    """
+    Get the particle velocity in the heliosphere frame.
 
     Parameters
     ----------
-    direct_events: dict
+    direct_events : iterable
         Dictionary of direct events.
 
     Returns
     -------
-    vultra_heliosphere_frame: np.ndarray
+    vultra_heliosphere_frame :
```

Comment:
```suggestion
    vultra_heliosphere_frame : np.ndarray
```

Were you running into errors with this one? I think to get this one working you may need to add an `import numpy as np` at the top of the file.